atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/DefaultClusterEventService.java

```diff
@@ -66,8 +64,8 @@
           new Builder()
               .register(Namespaces.BASIC)
               .register(MemberId.class)
-              .register(LogicalTimestamp.class)
-              .register(WallClockTimestamp.class)
+              .register(Void.class) // placeholder for the deleted LogicalTimestamp class
```

Comment:
:question: does kryo assign ids based on the order of registered types or something? Just wondering why we have to register Void here.